Formula/gtk4.rb

```diff
@@ -1,9 +1,9 @@
 class Gtk4 < Formula
   desc "Toolkit for creating graphical user interfaces"
   homepage "https://gtk.org/"
-  url "https://download.gnome.org/sources/gtk/4.4/gtk-4.4.1.tar.xz"
```

Comment:
The latest stable still shows `4.4.1`
> Latest stable v4.4.1

Reply:
![image](https://user-images.githubusercontent.com/1580956/149665150-e400d9d4-4bfc-4ed7-a374-b0974479f534.png)

Reply:
Huh, that's weird. The [tarball and changelogs for 4.6 are up](https://download.gnome.org/sources/gtk/4.6/) and the release is [tagged on their Gitlab](https://gitlab.gnome.org/GNOME/gtk/-/tree/4.6.0). I guess no-one's come round to right the announcement post and update the website yet. Would you mind keeping the PR open until that happens, probably next week?

Reply:
yeah, certainly, can you raise this to them to update their documentation page?

Reply:
I raised it with them on Twitter, but they also confirmed that 4.6.0 is officially out. https://twitter.com/GTKtoolkit/status/1483270411047608323

Reply:
Let's ship this then :) Thanks